hail/src/test/scala/is/hail/TestUtils.scala

```diff
@@ -473,12 +473,9 @@ object TestUtils {
     arrayElementsRequired: Boolean = true,
     skipInvalidLoci: Boolean = false,
     partitionsJSON: String = null): MatrixTable = {
-    val addedReference = rg.exists { referenceGenome =>
-      if (!ReferenceGenome.hasReference(referenceGenome.name)) {
-        ReferenceGenome.addReference(referenceGenome)
-        true
-      } else false
-    } // Needed for tests
+    rg.foreach { referenceGenome =>
+      ReferenceGenome.addReference(referenceGenome)
```

Comment:
I think the reason I added this was for the case where the random reference genome generator generated two different references with the same name. I think that's unlikely to happen, but it's possible a test could randomly fail because of this.

Reply:
Where do we generate random references?  Type generator for TLocus just samples from the existing references and I didn't see anything in ReferenceGenomeSuite.

Reply:
ReferenceGenome.gen -> VSMSubgen.random which is used in the following test suites: ImportVCFSuite, ExportVCFSuite, VSMSuite. It's also used in VSMSubgen.realistic which is not used anywhere.

Reply:
You already handled that!  It generates unique names:

 >     name <- Gen.identifier.filter(!ReferenceGenome.hasReference(_))

Reply:
Awesome! I can't remember what I added. It's been over a year since I worked on the ReferenceGenome.